VueVixens/docs
# 169-Workshop translations to Spanish

Partial Spanish translation for  full-day Vue workshop. 

**Feature**

- Dropdown language menu.

- Appendix_1.

- ch1.

- Vue intro.

- Add all images We would need in future for this workshop.

**Changes Proposed**

Partial Spanish translation for  full-day Vue workshop.  Dropdown language menu, appendix_1, ch1, vue intro. and  all images We would need in future for this workshop were added.

**Testing Instructions**

1. In Navbar you notice language dropdown, click on it, and then select Spanish, Languages label in navbar changes to Idioma and you find intro in spanish for workshops.

2. In vertical navbar in left side, you find two titles completely traduced: Ch1 and appendix_1.


**Extra Information**

I didnt know how to introduce more reviewers, but would be nice if @FlorIBG or our gals that speak spanish could take a look of this commit, thanks!
